tests_integration/test_procedure.py

```diff
@@ -22,67 +23,119 @@
 def test_snowpark_procedure_flow(
     _test_steps, temporary_working_directory_ctx, alter_snowflake_yml
 ):
-    _test_steps.assert_that_no_entities_are_in_snowflake()
+    _test_steps.assert_no_procedures_in_snowflake()
+    _test_steps.assert_no_functions_in_snowflake()
```

Comment:
I'm not sure about that idea, test is so long

Reply:
But it take less time because we reduce common steps. Also this is a single functionality so w test it holistically. We have separated units to do detailed testing of functions vs procedures.